test-data/unit/check-plugin-attrs.test

```diff
@@ -2321,3 +2321,55 @@ reveal_type(b.x)  # N: Revealed type is "builtins.int"
 reveal_type(b.y)  # N: Revealed type is "builtins.int"
 
 [builtins fixtures/plugin_attrs.pyi]
+
+[case testDefaultHashability]
+from typing import Hashable
+from attrs import define
+
+@define
+class A:
+    a: int
+
+reveal_type(A.__hash__)  # N: Revealed type is "None"
+
+[builtins fixtures/plugin_attrs.pyi]
+[typing fixtures/typing-full.pyi]
+
+[case testFrozenHashability]
+from typing import Hashable
+from attrs import frozen
+
+@frozen
+class A:
+    a: int
+
+a: Hashable = A(1)
```

Comment:
The same for all other tests

Reply:
Oops, done. I've also added an additional test for subclassing behavior.